coregistration/Coregistration.ipynb

```diff
@@ -240,156 +196,85 @@
+    "coregister_thunder = ThunderRegistrationTask(feat_bands, channel=2)\n",
+    "\n",
+    "eopatch_thunder = coregister_thunder(eopatch_clean)\n",
     "\n",
-    "eopatch_thunder = coregister_thunder(eopatch_clean)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Write result to GIF"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 14,
-   "metadata": {},
-   "outputs": [],
-   "source": [
     "make_gif(eopatch=eopatch_thunder.data[\"BANDS\"] * 255, project_dir=\".\", filename=\"eopatch_thunder.gif\", fps=3)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Enhanced Cross-Correlation in OpenCV"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "This algorithm uses intensity values to maximise cross-correlation between pair of images. It uses an Euler transformation (x,y translation plus rotation). "
+    "### Enhanced Cross-Correlation in OpenCV\n",
+    "\n",
+    "This algorithm uses intensity values to maximise cross-correlation between pair of images. It uses an Euler transformation (x,y translation plus rotation)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
     "params = {\"MaxIters\": 200}\n",
-    "coregister_ecc = ECCRegistration((FeatureType.DATA, \"BANDS\"), channel=2, params=params)\n",
     "\n",
-    "eopatch_ecc = coregister_ecc(eopatch_clean)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 16,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "coregister_ecc = ECCRegistrationTask(feat_bands, channel=2, params=params)\n",
+    "\n",
+    "eopatch_ecc = coregister_ecc(eopatch_clean)\n",
+    "\n",
     "make_gif(eopatch=eopatch_ecc.data[\"BANDS\"] * 255, project_dir=\".\", filename=\"eopatch_ecc.gif\", fps=3)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Point-Based Registration in OpenCV"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Three transformation models are supported for point-based registration, i.e. Euler, PartialAffine and Homography. These methods compute feature descriptors (i.e. SIFT or SURF) of the pair of images to be registered, and estimate a robust transformation using RANSAC to align the matching points. These methods perform poorly compared to the other methods due to the inaccuracies of the feature extraction, point-matching and model fitting. If unplausible transformations are estimated, a warning is issued and an identity matrix is employed instead of the estimated transform. Default parameters are (Model=Euler, Descriptor=SIFT, RANSACThreshold=7.0, MaxIters=1000).\n",
-    "\n",
-    "Note: In case the following cell will raise an error\n",
+    "### Point-Based Registration in OpenCV\n",
     "\n",
-    "```Python\n",
```

Comment:
Is there a reason this was removed?

Reply:
yes ( opencv-contrib-python == 4.6.0.66 ( and eo-learn 1.3.1 ))

Reply:
cool enough